src/every_query/aces_to_eq/aces_to_eq.py

```
@@ -0,0 +1,89 @@
+def code_slug(code: str, n_hash: int = 10, prefix_len: int = 24) -> str:
+    h = hashlib.sha1(code.encode("utf-8")).hexdigest()[:n_hash]
+    prefix = re.sub(r"[^A-Za-z0-9._-]+", "_", code).strip("_")[:prefix_len]
+    return f"{prefix}__{h}" if prefix else h
+
+
+def shard_id(p: Path) -> int:
+    return int(p.stem)
+
+
+def create_eq_task_df(
+    eq_shard_fp: str, aces_shard_fp: str, codes: list[str], output_root: str, target_rows: int
```

Ruff (0.14.14)

[warning] Unused function argument: `target_rows`

(ARG001)